_inc/client/scss/style.scss

```diff
@@ -13,6 +13,7 @@
 
 // Variables
 @import 'variables/colors';
+@import '../../../node_modules/@automattic/calypso-color-schemes/src/__color-studio/color-variables.scss';
```

Comment:
Remember to add `@automattic/calypso-color-schemes` to direct dependencies (it's now pulled in via `calypso-build`)

Reply:
Anything here that might end up unintentionally overwriting variables from `variables/colors`?

Reply:
Good point! I'll change the order so even if there are conflicts, the variables in `variables/colors` has precedence.